webpack/assets/javascripts/react_app/components/ToastsListRails/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { notify } from '../../../foreman_toast_notifications';
+import ToastsList from '../ToastsList';
+
+const ToastsListRails = ({ rails_messages: railsMessages }) => {
+  useEffect(() => {
+    railsMessages.forEach(({ message, type }) => {
+      notify({ message, type });
```

Comment:
this effect will trigger on the first mount and every update, even if there's no change 
should we skip this effect if `railsMessages` are the same?

Reply:
Won't the next line ` [railsMessages]);` prevent it from happening?

Reply:
no, it will be invoked only if `railsMessages` has changed.

Reply:
I'm not sure what are you saying is the wanted behavior here

Reply:
well... my bad. I missed the `[railsMessages]` :man_facepalming: